src/Presence/kahoot.json

```
@@ -94,5 +94,97 @@
+    },
+    "kahoot.onPodium": {
+        "message": "On the Podium",
+        "description": "Used in the context of a player/team being withing the top 3"
+    },
+    "kahoot.pollAnswer": {
+        "message": "Answered Poll",
+        "description": "Used when a player is viewing the results of a poll"
+    },
+    "kahoot.drumRoll": {
+        "message": "Waiting for Final Ranking...",
+        "description": "Used when a player is waiting to receive their final ranking"
+    },
+    "kahoot.position": {
+        "message": "Position: {0}",
+        "description": "Used to display the position (ranking) of a player. {0} is the position"
+    },
+    "kahoot.teamTalk": {
+        "message": "Discussing with Tean...",
+        "description": "Used when a player is talking to their team during team talk time"
+    },
+    "kahoot.gameLocked": {
+        "message": "Game Locked",
+        "description": "Used when a game is locked"
+    },
+    "kahoot.gameSummary": {
+        "message": "Looking at Game Summary...",
+        "description": "Used when the host is viewing the summary of a game"
+    },
+    "kahoot.login": {
+        "message": "Logging in...",
+        "description": "Used when a user is logging in"
+    },
+    "kahoot.createHome": {
+        "message": "Viewing Create Home Page...",
+        "description": "Used when the user is viewing the create home page"
+    },
+    "kahoot.discover": {
+        "message": "Viewing Kahoot! Discover Page...",
+        "description": "Used when the user is viewing the discover page"
+    },
+    "kahoot.searchKahoots": {
+        "message": "Searching for Kahoots:",
+        "description": "Used when the user is searching for Kahoots"
+    },
+    "kahoot.kahootDetails": {
+        "message": "Viewing Kahoot Details:",
+        "description": "Used when the user is viewing the details of a Kahoot"
+    },
+    "kahoot.kahootProfile": {
+        "message": "Viewing Kahoot Profile:",
+        "description": "Used when the user is viewing the profile of a Kahoot"
+    },
+    "kahoot.myKahoots": {
+        "message": "Viewing My Kahoots...",
+        "description": "Used when the user is viewing their own kahoots"
+    },
+    "kahoot.userReports": {
+        "message": "Viewing Game Reports...",
+        "description": "Used when the user is viewing their game reports"
+    },
+    "kahoot.myCourses": {
+        "message": "Viewing My Courses...",
+        "description": "Used when the user is viewing their own courses"
+    },
+    "kahoot.editingCourse": {
+        "message": "Editing a Course:",
+        "description": "Used when the user is editing a course"
+    },
+    "kahoot.viewingCourse": {
+        "message": "Viewing a Course:",
+        "description": "Used when the user is viewing a course"
+    },
+    "kahoot.editingKahoot": {
+        "message": "Editing a Kahoot:",
```

ditto

---

why was this resolved?
```suggestion
        "message": "Editing Kahoot:",
```

---

Should be correct now.